File: library/src/main/java/com/alamkanak/weekview/EventLongPressListener.java
package com.alamkanak.weekview;

import android.graphics.RectF;

public interface EventLongPressListener {
    /**
     * Similar to {@link com.alamkanak.weekview.WeekView.EventClickListener} but with a long press.
     *
     * @param event:     event clicked.
     * @param eventRect: view containing the clicked event.
     */
    void onEventLongPress(WeekViewEvent event, RectF eventRect);
}

File: library/src/main/java/com/alamkanak/weekview/EventLongPressListener.kt
package com.alamkanak.weekview

import android.graphics.RectF

interface EventLongPressListener {
    /**
     * Similar to [com.alamkanak.weekview.EventClickListener] but with a long press.
     *
     * @param event:     event clicked.
     * @param eventRect: view containing the clicked event.
     */
    fun onEventLongPress(event: WeekViewEvent, eventRect: RectF)
}
